ras/ras_ppcutils.py

```diff
@@ -447,3 +449,40 @@ def test_lparnumascore(self):
         for list_item in lists:
             self.run_cmd('lparnumascore %s' % list_item)
         self.error_check()
+
+    @skipIf(IS_POWER_NV or IS_KVM_GUEST,
+            "This test is not supported on KVM guest or PowerNV platform")
+    def test_validate_lparstat(self):
+        """
+        Test case validates laprstat -E output %busy and %idle
+        should not be < 0 or > 100
+        Normalized %busy + %idle should be equal to percentage under frequency
+        """
+        output = process.system_output("lparstat -E 1 1").decode("utf-8")
+        input_string = ''
+        for line in output.splitlines():
+            if 'GHz' in line:
+                input_string = line
+                # Define the regular expression pattern
+                pattern = r'(\d+\.\d+)\s+(\d+\.\d+)\s+\d+\.\d+GHz\[\s*(\d+)%\]\s+(\d+\.\d+)\s+(\d+\.\d+)'
```

Comment:
@shirishaganta1 looks like the length is >79. please take care of it.

Reply:
sure will take care of that